objectscale-manager/crds/storagepolicy.yaml

```
@@ -0,0 +1,25 @@
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: storagepolicies.ecs.dellemc.com
```

we need to rename this,  it conflicts with storagepolicies on vsphere.

```
─ $ ▶  kubectl get storagepolicies
NAME                                    AGE
dellemc-objectscale-highly-available    13h
dellemc-objectscale-vsan-direct-thick   13h
dellemc-objectscale-vsan-sna-thick      13h

```

---

fixed

---

but this name is still storagepolicies...  which kinda conflicts with vsphere storagepolicies...  
```suggestion
  name: storagepolicies.ecs.dellemc.com
```

---

However we have group ECS

---

is it ok for you?